mailer.js

```diff
@@ -26,6 +27,19 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocalTimeString (timeZone, locale) {
+    // if no timeZone is passed, use 'Etc/UTC'
+    moment.tz.setDefault('Etc/UTC')
+    // if no locale is passed, use 'en'
+    locale = locale || 'en'
+    moment.updateLocale(locale)
```

Comment:
[According to the moment docs, updateLocale](http://momentjs.com/docs/#/i18n/) is to `to update a locale's properties.` This does not change the locale used for translations. Instead `moment.locale` must be used. When I test with Spanish, updateLocale always prints the date in English. When I use `update`, Spanish is used.

## with updateLocale date is in English

<img width="338" alt="screen shot 2016-07-22 at 11 17 14" src="https://cloud.githubusercontent.com/assets/848085/17054230/fe30120c-4ffd-11e6-83df-d05ea26529be.png">

## with update, it's in Spanish

<img width="364" alt="screen shot 2016-07-22 at 11 17 47" src="https://cloud.githubusercontent.com/assets/848085/17054237/079c6930-4ffe-11e6-920e-c2e82ed31e17.png">